students/Roy_Tate/lesson02/series.py

```diff
@@ -0,0 +1,124 @@
+# author: githubtater
+
+def fibonacci(n):
+    '''Return the nth value in the fibonacci series.'''
```

Comment:
per pep8 use `"""triple double quotes""" ` for doc strings 

https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring